docs/getting-started.md

```diff
@@ -0,0 +1,83 @@
+## **Getting Started**
+
+Please use the following outline to get started with a new Data Science Project. If you run into any problems or need additional support, please join our [slack](https://operatefirst.slack.com/ssb/redirect) and post a message in the `#support` channel, or open an issue [here](https://github.com/operate-first/support/issues).  
+
+### **Define a Project Document**
+
+1. Review our project workflow document [here][1].
+
+2. Create a new project description using our [template][2]. For reference, you can review an example project document [here][3].
+
+3. Put your project document into a subfolder of this [shared directory][4]. 
+
+### **Create Git Repository from a Template**
+
+Next, create a [CookieCutter][5] formatted data science project repository from this [template][6]. To do so, please follow these [instructions][7] for creating a new repository from a template.
+
+### **Set Up AICoE Continuous Integration (CI)**
+
```

Comment:
I'm not sure how https://www.operate-first.cloud/data-science/ocp-ci-analysis/ can help with setting up the thoth bots, prow and other aspects of AICoE CI here

Reply:
Got it, would you like us to include set up for thoth bots and prow (and any other aspects) here?

Reply:
@amsaparov let's start with linking the document. We can eventually work with the thoth team to have a detailed guide or even have an espresso series video on this! See my following suggestion